src/collapsible.tab.js

```diff
@@ -68,14 +68,14 @@
 				e.preventDefault();
 				e.stopPropagation();
 
-				if( self.$tabHeader.is( '.' + activeTabClass ) ) {
-					deactivateTab( self.$tabHeader );
-				} else {
-					deactivateTab( $tabContainer.find( '.' + activeTabClass ) );
+				if( !self.$tabHeader.is( '.' + activeTabClass ) ) {
+					var nonActives = $tabContainer.find( '.' + activeTabClass ).not( self.$tabHeader );
+					if( nonActives.length ){
+						deactivateTab( nonActives );
+					}
 					activateTab( self.$tabHeader );
+					self.expand();
```

Comment:
I think the calls to the empty set of elements in our library and jQuery are probably pretty cheap. You can probably just compress this to:

```js
deactiveTab( actives );
activateTab( self.$tabHeader );
```

Reply:
agreed on all edits thanks john